vendor.mod

```diff
@@ -5,34 +5,36 @@
 
 module github.com/docker/docker
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.2
```

Comment:
needs to be removed similar to https://github.com/moby/buildkit/pull/5507#discussion_r1860277406